Bottom: 0ef02070f7597811030d4b610d27fcf7793a5e54
Top:    7cb0bb63ab8fce10c5cdbb6f790a3da5f4479116
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 21:23:47 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
